sci-astronomy/pal/pal-0.9.8-r1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Positional Astronomy Library"
+HOMEPAGE="https://github.com/Starlink/pal"
+SRC_URI="https://github.com/Starlink/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="static-libs"
+RDEPEND="sci-astronomy/erfa:="
+DEPEND="${RDEPEND}"
```

Comment:
Shouldn't this depend on `dev-libs/libbsd` then? Since this is wat provides `bsd/string.h`.

Reply:
Even with dev-libs/libbsd I can repro due the error. From what it looks like, HAVE_BSD_STRING_H is not properly set by the configure script hence the inclusion fails/doesn't happen. I might be wrong.

Reply:
Right, but if the header is not there the build will fail as well, so if we patch it like this we have to DEPEND on libbsd.

But we only need this when building with clang/musl right? This way we also change the build if we compile with gcc, probably what we want then instead is to set HAVE_BSD_STRING_H if the compiler is clang. Though then still we have to DEPEND on libbsd so maybe this is not worth the extra effort.

Reply:
Aha ... got it. How does it look now?